src/app/static/src/tests/components/runReport/changeLog.test.ts

```diff
@@ -0,0 +1,69 @@
+import {mount} from "@vue/test-utils";
+import ChangeLog from "../../../js/components/runReport/changeLog.vue";
+
+describe(`changeLog`, () => {
+    const changelogTypeOptions = ["internal", "public"]
+
+    const changelogStyle = {
+        label: {size: 2, justify: "text-right"},
+        control: {size: 6}
+    }
+    const getWrapper = () => {
+        return mount(ChangeLog,
+            {
+                propsData: {
+                    changelogStyle: changelogStyle,
+                    showChangelog: true,
+                    changelogTypeOptions: changelogTypeOptions
+                }
+            })
+    }
+
+    it(`renders component as expected`, () => {
+        const wrapper = getWrapper();
+        expect(wrapper.find("#changelog-message").exists()).toBeTruthy()
+        expect(wrapper.find("#changelog-type").exists()).toBeTruthy()
+        expect(wrapper.vm.$data).toEqual({"changeLogMessageValue": "", "changeLogTypeValue": "internal"})
+        expect(wrapper.vm.$props).toEqual(
+            {
+                changelogTypeOptions,
+                changelogStyle,
+                "showChangelog": true
+            })
+    })
+
+    it(`does not render component if showChangeMessage is false`, async() => {
```

Comment:
This test and some of the others now effectively [duplicate](https://github.com/vimc/orderly-web/blob/mrc-2311_git/src/app/static/src/tests/components/runReport/runReport.test.ts#L571) those elsewhere. The tests in `runReport.test.ts` that only test `<change-log>` and are therefore independent of `<run-report>` should be removed in favour of these new ones as the duplication means more to maintain.